runtime/command/src/main/java/io/aklivity/zilla/runtime/command/internal/ZillaMain.java

```diff
@@ -53,9 +54,15 @@ private static int invoke(
                     .withCommand(Help.class);
 
             ClassLoader loader = Thread.currentThread().getContextClassLoader();
+            boolean incubatorEnabled = incubatorEnabled();
+
             for (ZillaCommandSpi service : ServiceLoader.load(ZillaCommandSpi.class, loader))
             {
-                service.mixin(builder);
+                if (!service.getClass().getPackageName().contains("incubator") ||
+                    incubatorEnabled)
+                {
+                    service.mixin(builder);
+                }
```

Comment:
Use `FeatureLoader.load(...)` instead, so no need for if condition around service.mixin(...) call.